code/API_definitions/qod-api.yaml

```diff
@@ -455,7 +464,7 @@ components:
             $ref: "#/components/schemas/ErrorInfo"
           example:
             code: NOT_FOUND
-            message: "Session Id does not exist"
+            message: "Session Id does not exist"     
```

Comment:
What is the change here? Is it needed?

Reply:
**There is No any Changes there.** I just removed Generic501 schema from  below that line there might  be some "Indentation or spaces removed"

Reply:
There seem to be 3-4 extra spaces. It cosmetic but would be easy to get rid of them.